src/main/java/uk/gov/companieshouse/lookup/model/CompanyConfirmation.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.companieshouse.lookup.model;
+
+import java.time.LocalDate;
+
+public class CompanyConfirmation {
```

Comment:
Maybe worth renaming to something like `CompanyDetails` since the 'Confirmation' could be a little misleading